woocommerce_fusion/tasks/sync_sales_orders.py

```
@@ -23,6 +23,26 @@
+		reverse = {v: k for k, v in mapping.items()}
+		return mapping, reverse
+	except Exception:
+		return dict(WC_ORDER_STATUS_MAPPING), dict(WC_ORDER_STATUS_MAPPING_REVERSE)
+
+
+def _get_allowed_inbound_statuses(woocommerce_server: str) -> list[str]:
+	"""Use server method to get allowed inbound statuses with default fallback."""
+	try:
+		wc_server = frappe.get_cached_doc("WooCommerce Server", woocommerce_server)
+		return wc_server.get_allowed_inbound_statuses()
+	except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)